generator/config.js

```diff
@@ -7,6 +7,7 @@ const explorer = cosmiconfig("mst-gql")
 const defaultConfig = {
   excludes: [],
   force: false,
+  keepInternalOrder: false,
```

Comment:
what does this option do?

Reply:
It reads in the existing `internal.js/s` file and how things are ordered in it and then produces a new `internal.js/ts` file that tries to keep modules in that same order (prepending new ones).

This very easily allows you to resolve circular import dependencies. I've put it behind a flag so that it doesn't break for people not using the feature because suddenly it will prepend any new modules that are added later (which is almost certainly wrong since they get imported before BaseModel)

Reply:
maybe I'm still confused, but because this is a generated file can't we just always import things in an ideal order and forgo adding an option?